test/suite/test_backup18.py

```diff
@@ -0,0 +1,135 @@
+#!/usr/bin/env python
+#
+# Public Domain 2014-2020 MongoDB, Inc.
+# Public Domain 2008-2014 WiredTiger, Inc.
+#
+# This is free and unencumbered software released into the public domain.
+#
+# Anyone is free to copy, modify, publish, use, compile, sell, or
+# distribute this software, either in source code form or as a compiled
+# binary, for any purpose, commercial or non-commercial, and by any
+# means.
+#
+# In jurisdictions that recognize copyright laws, the author or authors
+# of this software dedicate any and all copyright interest in the
+# software to the public domain. We make this dedication for the benefit
+# of the public at large and to the detriment of our heirs and
+# successors. We intend this dedication to be an overt act of
+# relinquishment in perpetuity of all present and future rights to this
+# software under copyright law.
+#
+# THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND,
+# EXPRESS OR IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF
+# MERCHANTABILITY, FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT.
+# IN NO EVENT SHALL THE AUTHORS BE LIABLE FOR ANY CLAIM, DAMAGES OR
+# OTHER LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE,
+# ARISING FROM, OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR
+# OTHER DEALINGS IN THE SOFTWARE.
+
+import wiredtiger, wttest
+import os, shutil
+from helper import compare_files
+from suite_subprocess import suite_subprocess
+from wtdataset import simple_key
+from wtscenario import make_scenarios
+
+# test_backup18.py
+# Test backup:query_id API.
+class test_backup18(wttest.WiredTigerTestCase, suite_subprocess):
+    conn_config= 'cache_size=1G,log=(enabled,file_max=100K)'
+    mult=0
+    nops=100
+    pfx = 'test_backup'
+    uri="table:test"
+
+    def id_check(self, expect):
+        got = []
+        bkup_c = self.session.open_cursor('backup:query_id', None, None)
+        while True:
+            ret = bkup_c.next()
+            if ret != 0:
+                break
+            idstr = bkup_c.get_key()
+            got.append(idstr)
```

Comment:
I _think_ this can be reduced to:
```
for idstr in bkup_c:
   got.append(idstr)
```

Reply:
We cannot use that for backup cursors because they don't return values. I think one or more of the backup tests have that as a comment, such as `test_backup15.py`. I will add that comment here too.